src/classes/CRLP_RollupBatch_SVC.cls

```diff
@@ -64,41 +64,16 @@ public class CRLP_RollupBatch_SVC {
         String queryFilter = '';
         Integer maxRelatedOppsForNonLDVMode = CRLP_Rollup_SVC.getMaxRelatedOppsForNonSkewMode();
 
-        // If the Summary Object is the Account, then filter on Accounts that have at least a single
-        // Opportunity attached. This is helpful to reduce the overall query size.
-        // To handle a scenario where an attached Opportunity was deleted, but the record not recalculated
-        // also include any records where the TotalGifts or TotalMemberships fields are not zero.
-        // If the field is null, a query with "< #" will not work. "= null" must be explicitly included in the query.
         if (jobType == CRLP_RollupProcessingOptions.RollupType.ContactHardCredit ||
-                jobType == CRLP_RollupProcessingOptions.RollupType.AccountHardCredit) {
+            jobType == CRLP_RollupProcessingOptions.RollupType.AccountHardCredit ||
+            jobType == CRLP_RollupProcessingOptions.RollupType.ContactSoftCredit ||
```

Comment:
Of the different types of rollup jobs, I believe the goal will be to use the new `CustomizableRollups_UseSkewMode__c` field for all jobs except for the GAU and RD jobs. If true, I think this condition might be easier as just `jobType != .GAU and jobType != .RecurringDonations`. That would allow the use of the new condition for all Contact and Account targeted jobs.